app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -26,6 +26,11 @@ open class BrowserApplication : Application() {
         setupLogging(megazordEnabled)
     }
 
+    override fun onTrimMemory(level: Int) {
+        super.onTrimMemory(level)
+        components.core.sessionManager.onLowMemory()
```

Comment:
We should limit calling this to the main process, see here:
https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt#L112

Otherwise we end up initializing the world in other processes too. In the worst case we do that in a gecko child process and re-create a gecko runtime there which will blow up. :)

Reply:
Done!